great_expectations/validator/validator.py

```diff
@@ -414,7 +414,9 @@ def __getattr__(self, name):
         if self.active_batch is None:
             raise TypeError("active_batch cannot be None")  # noqa: TRY003
         name = name.lower()
-        if name.startswith("expect_") and get_expectation_impl(name):
+        if (
+            name.startswith("expect_") or name == "unexpected_rows_expectation"
+        ) and get_expectation_impl(name):
```

Comment:
This is only needed for the test runner as calling `Validator.unexpected_rows_expectation` is no longer part of the public API.